src/Formatter/PhpcsFormatter.php

```diff
@@ -24,7 +24,7 @@ public function format(Process $process): string
             return $process->getErrorOutput();
         }
 
-        $pos = strrpos($output, "\n");
+        $pos = strrpos(trim($output), "\n");
```

Comment:
Wouldn't it be better to do this trim at line 22 instead?
That way, the code always works with the normalized version.
(Now $pos works on a different output value then the substr method)

Reply:
This patch works for me, where I have the same "rogue" JSON output. I'm using 0.16.1

Reply:
Until this is patched, we can run the following:

```
sed -i 's#$pos = strrpos($output, "\\n");#$pos = strrpos(trim($output), "\\n");#g' vendor/phpro/grumphp/src/Formatter/PhpcsFormatter.php
```